### PR TITLE
Fixes #32613 - Update error message on proxy

### PR DESCRIPTION
**Pull request by sjha4**

### CODE REVIEW

File: app/lib/katello/errors.rb [hunk 1]
```diff
@@ -53,7 +53,7 @@ def message
 
     class PulpcoreMissingCapabilities < StandardError
       def message
-        _("A smart proxy seems to have been refreshed without pulpcore being running.  You may want to ")
+        _("A smart proxy seems to have been refreshed without pulpcore being running.  You may want to refresh your smart proxy.")
```
Comment (jeremylenz) [inline app/lib/katello/errors.rb:56]: Will refreshing it again help?  How can the problem and the solution be the same? Do I need to make sure pulpcore is running first?

Comment (sjha4) [reply app/lib/katello/errors.rb:56]: So this is thrown when pulp3 is enabled on the proxy and yet PULP3 as a feature is not found on the proxy. There could be an error on the proxy with the pulp services running that led to this. I believe in most conditions it is caused by a race condition during installation. @jlsherrill can correct me on that.

We could say something like, 'Please refresh the smart proxy after ensuring that pulpcore is running'.

Comment (jeremylenz) [reply app/lib/katello/errors.rb:56]: > Please refresh the smart proxy after ensuring that pulpcore is running.

yes this makes more sense to me!

Comment (jlsherrill) [reply app/lib/katello/errors.rb:56]: yes exactly, the installer couldn't guarantee that pulp was fully up when the smart proxy was being initially refreshed.  We try to detect this situation and run a refresh manually, but this is kinda a last resort suggestion

Comment (sjha4) [reply app/lib/katello/errors.rb]: Updated it to say:
`A smart proxy seems to have been refreshed without pulpcore being running. Please refresh the smart proxy after ensuring that pulpcore services are running.`

**Review by jeremylenz — APPROVED** [review]

**Pull request merged**